src/test/java/com/bereda/service/CurrencyServiceTest.java

```diff
@@ -145,34 +145,32 @@ void shouldThrowExceptionWhenDoNotFindExchangeRateValueFromEurToPln() {
 
 
     public static class Fixtures {
-
-
         Currency eurToPlnData = createFakeEurToPlnData();
         Currency plnToEurData = createFakePlnToEurData();
         String from = "EUR";
         String to = "PLN";
         Double rateEurToPln = 4.00;
         LocalDate localDate = LocalDate.of(2020, 12, 12);
-
+        LocalDateTime localDateTime = LocalDateTime.of(2020,12,12,5,55);
 
         private Currency createFakeEurToPlnData() {
```

Comment:
instead of createFakeEurToPlnData and createFakePlnToEurData you can create just createFakeCurrency method that accepts from, to, value and createdAt and call it with different params to define plnToEurData and eurToPlnData